### ISSUE TITLE
BUG: Max Retires exceeded with file requests

### ISSUE THREAD

**Issue by csmcallister** [GSA/srt-fbo-scraper#156]
## Expected Behavior
If you're able to visit a url manually and download a file, python `requests` should be able to do so as well.

## Current Behavior
A request failed after max retries were exceeded (0 by default in requests), likely because we're sending too many requests from same ip address in short period of time.

Partial traceback:
```
HTTPSConnectionPool(host='www.neco.navy.mil', port=443): Max retries exceeded with 
url:
 /upload/n61340/n6134019q0015combined_synopsis-panasonic_toughbooks.docx 
(Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 
0x7f211fd09b70>: Failed to establish a new connection: [Errno 110] Connection 
timed out',))"
```

## Possible Solution
Retry urls with  `requests.packages.urllib3.util.retry.Retry`, which has many options to simplify retries.

## Context
It's important to download all attachments related to a solicitation as we don't want to miss any that have 508-related language.

**Comment by csmcallister** [comment]
likely fixed by 9efe2dc, but will wait until tomorrow's run to verify in logs.

**Comment by csmcallister** [comment]
fixed by 9efe2dc
